ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -1597,6 +1650,7 @@ const triggerSet: TriggerSet<Data> = {
           fr: 'AoE \'x\' sur VOUS',
           ja: '自分に\'x\'',
           cn: '\'x\' 点名',
+          tc: '\',
```

Comment:
It looks like the escape character here caused an issue. In this there the x is quoted as it is a literal description of the marker, 'x'.